pinner/pinning_op/pinning_op.go

```diff
@@ -0,0 +1,77 @@
+package pinning_op
```

Comment:
nit: `pinning_op` -> `operation` for idiomatic package name and so it can be imported as `/pinner/operation` and renamed locally as oneof: `op` or `po` or `pinop`

also should update folder and filename to operation